modules/data.remote/R/MODIS_LC_prep.R

```diff
@@ -0,0 +1,178 @@
+#' Prepare MODIS land cover data for the SDA workflow.
+#'
+#' @param site_info Bety list of site info including site_id, lon, and lat.
+#' @param time_points A vector contains each time point within the start and end date.
+#' @param outdir Where the final CSV file will be stored.
+#' @param export_csv Decide if we want to export the CSV file.
```

Comment:
these two tags seem redundant. If outdir == NULL then you don't want to export the CSV. That said, I'd expand the explanation text to make that point explicit (i.e. function returns data object by default, outdir is NULL by default, but if you want to export to a CSV you specify the path here)

Reply:
Fixed.